schemas/expconf/v0/searcher-custom.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://determined.ai/schemas/expconf/v0/searcher-custom.json",
+    "title": "CustomConfig",
+    "type": "object",
+    "additionalProperties": false,
+    "required": [
```

Comment:
looks good to me. Do we need anything else here? Or we can add it later if we can think of anything.

Reply:
exactly - I cannot think of anything right now but we are free to update this while the feature branch is there.